setup.py

```diff
@@ -99,29 +101,31 @@ def get_version(version_file):
         "pytz>=2018.5",
         "requests>=2.18.4",
         # This is necessary to support the python_requires kwarg
```

Comment:
If we are no longer pinning `setuptools` I don't know that this comment explaining the pin is necessary.

```suggestion
```